frontend/rolecall/src/app/piece/piece_editor.component.ts

```diff
@@ -526,4 +545,13 @@ export class PieceEditor implements OnInit {
     }
     return code;
   }
+
+  toggleOpen(index: number) {
+    const superBallet = this.displayedPieces[index];
+    if (superBallet.type == 'SUPER') {
+      superBallet.isOpen = !superBallet.isOpen;
+      this.pieceAPI.setPiece(superBallet);
```

Comment:
Why does visibility state need to be managed on the API layer? It seems like it's a presentational concern that the API layer shouldn't know about. Also this is not something that should be sent to the backend as it will slow down UI interactions.

Reply:
This has already been removed from the sibling PR.